### DISCUSSION TITLE
Multi UAV flight with Tello

### DISCUSSION THREAD

**Discussion by lidiaxp** [ctu-mrs/mrs_uav_system#178] [General]
I am trying to do the real flights with Tello using the MRS System. I see you are using the TelloPy to wrapper Tello with the MRS system. Is it possible to change just the session.yml to add several Tellos? I know it is possible with the MRS's UAVs, but for Tello I could not figure out how to do it yet, mainly because the TelloPy use a socket communication, and when I was reading it looked like it is using a fixed IP. Which file should I define the IP if it is possible?

Thank you for your help

**Comment by klaxalk** [comment]
Hey Lidia, there is no limitation on the side of the MRS UAV System. But how are you connecting multiple tellos to a single PC? Since each tello typically creates an access point and presents itself using a static IP (the same for all tellos).

Anyway, the IP in the tello interface is defined here https://github.com/ctu-mrs/mrs_uav_dji_tello_api/blob/6c424901b675fd1039f0c72e83df2b0cb277709d/scripts/tellopy/_internal/tello.py#L64

It could be modified from within my wrapper after here: https://github.com/ctu-mrs/mrs_uav_dji_tello_api/blob/6c424901b675fd1039f0c72e83df2b0cb277709d/scripts/tellopy_wrapper.py#L85 by e.g. doing 
```python
        self.tello.tello_addr =  ...
```

And this, the tellopy_wrapper.py could be parametrized to change the IP from the launch file.

Feel free to make the changes and submit PR.

**Comment by lidiaxp** [comment]
Hey, thank you very much. 

A friend said he knows how to connect multiple Tellos, so I would like to try to do it using the MRS system. We are going to try it in the next days. If it works, I will submit a PR to change the IP and explaining how to connect with more than one Tello.
